### DISCUSSION TITLE
File Upload via uWebSockets

### DISCUSSION THREAD

**Discussion by westsuhanic** [uNetworking/uWebSockets#1284] [General]
Hello

I have a file upload websocket server implemented using nodejs and ws . I want to replicate this functionality using uWebSockets.

I have looked through the example code. I am not sure how to proceed using the existing examples on how to upload a file from a web browser client to a server implemented using uWebSockets.

Any help is greatly appreciated.

thank you,

west suhanic

**Comment by epsilon-0** [comment]
Coincidentally, I have a small app made to do just this - https://github.com/PurritoBin/PurritoBin
Example to upload a paste (the js can be easily modified to send a file instead of text) - https://bsd.ac/
